jenkins/bootstrap_test.py

```diff
@@ -2285,7 +2286,16 @@ def test_config_is_sorted(self):
                 separators=(',', ': ')
                 ) + '\n'
             if original != expect:
-                self.fail('config.json is not sorted, please run jobs/config_sort.py')
+                self.fail('jobs/config.json is not sorted, please run '
+                          '`bazel run //jobs:config_sort`')
+        with open(bootstrap.test_infra('prow/config.yaml')) as fp:
+            original = fp.read()
+            sys.path.insert(1, os.path.join(sys.path[0], '../jobs'))
```

Comment:
this is kinda gross.
can we just make a new test in `jobs/` instead?

maybe move this and the config.json test out of bootstrap_test.py, since neither belongs here.